### ISSUE TITLE
Question: Does Azurite blob storage support range download?

### ISSUE THREAD

**Issue by evanxg852000** [Azure/azure-sdk-for-rust#936]
When trying to download a byte range from Azurite, a `FeatureNotSupported` error is reported as follows.
Also, the integration test does not include range download, adding one fails with the same error.
```bash
Err(Error { context: Full(Custom { kind: HttpResponse { status: InternalServerError, error_code: Some("FeatureNotSupported") }, error: HttpError { status: InternalServerError, error_code: Some("FeatureNotSupported"), headers: {"x-ms-error-code": "FeatureNotSupported", "connection": "keep-alive", "server": "Azurite-Blob/3.18.0", "transfer-encoding": "chunked", "keep-alive": "timeout=5", "x-ms-request-id": "b7846c1d-1b22-482e-a7a6-c8e5ed157758", "date": "Thu, 14 Jul 2022 14:27:15 GMT", "content-type": "application/xml"}, body: b"<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<Error>\n  <Code>FeatureNotSupported</Code>\n  <Message>x-ms-range-get-content-crc64 header or parameter is not supported in Azurite strict mode. Switch to loose model by Azurite command line parameter \"--loose\" or Visual Studio Code configuration \"Loose\". Please vote your wanted features to https://github.com/azure/azurite/issues\nRequestId:b7846c1d-1b22-482e-a7a6-c8e5ed157758\nTime:2022-07-14T14:27:15.672Z</Message>\n</Error>" } }, "server returned error status which will not be retried") })
```

**Comment by evanxg852000** [comment]
Closing as it's resolved here https://github.com/Azure/Azurite/issues/1602
